client/src/libshared/FluidExpression.res

```diff
@@ -219,6 +229,10 @@ let isEmpty = (expr: t): bool =>
   | ERecord(_, list{}) => true
   | ERecord(_, l) => l |> List.filter(~f=((k, v)) => k == "" && !isBlank(v)) |> List.isEmpty
   | EList(_, l) => l |> List.filter(~f=\"<<"(not, isBlank)) |> List.isEmpty
```

Comment:
What's the deal with "<<" here, @pbiggar? Seeing if it makes sense for the Tuple case.

Reply:
This in an infix function in rescipt. In OCaml/FSharp, this would be:

```
(not << idBlank)
```

Which is function composition. I think we should remove uses of `"<<"`, which aren't super clear anyway. This one would be:

```
(e => not(isBlank(e)))
```